### PR TITLE
Removes React imports

### PR DESCRIPTION
**Pull request by emmaielle**
#### :page_facing_up: Description:

Removes all React imports, according to new JSX Transform
https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html

---

#### :heavy_check_mark: Tasks:

- Updated Eslint rules to ignore not having these imports
- Ran `npx react-codemod update-react-imports`, as seen [here](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#removing-unused-react-imports)

---

#### :play_or_pause_button: Demo:

https://www.loom.com/share/91e3a1f785874ff19b3823eda0559df7

@loopstudio/react-devs

### CODE REVIEW

File: .eslintcache [hunk 1]
```diff
@@ -0,0 +1 @@
...late/src/testUtils/mocks/auth.js",[],"/Users/moira/Projects/react-app-boilerplate/src/testUtils/mocks/base.js",[],"/Users/moira/Projects/react-app-boilerplate/src/theme.js",[],{"ruleId":"477","severity":1,"message":"478","line":3,"column":1,"nodeType":"479","endLine":3,"endColumn":25},{"ruleId":"480","severity":1,"message":"481","line":19,"column":7,"nodeType":"482","messageId":"483","endLine":19,"endColumn":20},{"ruleId":"477","severity":1,"message":"484","line":1,"column":1,"nodeType":"479","endLine":49,"endColumn":3},{"ruleId":"477","severity":1,"message":"484","line":1,"column":1,"nodeType":"479","endLine":49,"endColumn":3},{"ruleId":"477","severity":1,"message":"484","line":11,"column":1,"nodeType":"479","endLine":13,"endColumn":3},{"ruleId":"477","severity":1,"message":"485","line":4,"column":1,"nodeType":"479","endLine":11,"endColumn":3},"import/no-anonymous-default-export","Assign array to a variable before exporting as module default","ExportDefaultDeclaration","no-console","Unexpected console statement.","MemberExpression","unexpected","Assign object to a variable before exporting as module default","Assign arrow function to a variable before exporting as module default"]
```
Comment (lucaleivaloop) [inline .eslintcache:1]: Do we want this file or should we add it to `.gitignore`?

File: src/index.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
-import React, { StrictMode } from 'react';
+import { StrictMode } from 'react';
+import * as React from 'react';
```
Comment (lucaleivaloop) [inline src/index.js:2]: Maybe this can be solved in one line, maybe it was fine as it was?

Comment (emmaielle) [reply src/index.js:2]: probably as it was

**Review by elestu — APPROVED** [review]
LGTM. Should we add an ESLint rule to warn us about this?

**Something like this:**

`.eslintrc`:
```json
"no-restricted-imports": ["warn", {
  "paths": [
    {
      "name": "react",
      "importNames": ["default"]
    }
  ]
}]
```

`src/index.js`:
```js
import React, { StrictMode } from 'react'; // eslint-disable-line no-restricted-imports
```

**Review by mconiglio — APPROVED** [review]

**Review by ValeFranchi — APPROVED** [review]

**Review by mwolman — APPROVED** [review]

**Review by lucaleivaloop — APPROVED** [review]

**Pull request merged**